readme.org

```diff
@@ -568,26 +568,23 @@ On each hand, combo the primary and secondary thumb keys to emulate the tertiary
 
 *** Mouse Keys
 
-[[https://zmk.dev/docs/behaviors/mouse-emulation][ZMK supports mouse buttons only]].
+[[https://zmk.dev/docs/behaviors/mouse-emulation][ZMK supports mouse button keys, move and scroll]].
 
-**** Mouse Keys on Host
+**** Mouse Keys without ~CONFIG_ZMK_POINTING~
 
 Mouse movement requires [[https://en.wikipedia.org/wiki/Mouse_keys][enabling mouse keys on the host]].  Mouse scroll is not supported.
 
 - [[https://linuxreviews.org/HOWTO_use_the_numeric_keyboard_keys_as_mouse_in_XOrg][X11]]
 - [[https://support.apple.com/en-au/guide/mac-help/mh27469/mac][Mac]]
 - [[https://support.microsoft.com/en-us/windows/use-mouse-keys-to-move-the-mouse-pointer-9e0c72c8-b882-7918-8e7b-391fd62adf33][Windows]]
 
+**** Mouse support with ~CONFIG_ZMK_POINTING=y~
```

Comment:
This block should be deleted, since mouse keys on the host is no longer required, and support is removed in this PR.

Reply:
Sorry I meant only the selected lines. Please include your building section from https://github.com/manna-harbour/miryoku_zmk/blob/bad800d703b710df475bd56e5b98c355700a551e/readme.org?plain=1#L583 to https://github.com/manna-harbour/miryoku_zmk/blob/bad800d703b710df475bd56e5b98c355700a551e/readme.org?plain=1#L589

Reply:
I have reverted selected lines. Is it okay now?